src/fedora/39/amd64/Dockerfile

```diff
@@ -29,8 +29,18 @@ RUN dnf --setopt=install_weak_deps=False install -y \
         tar \
         procps \
         zip \
+        azure-cli \
     && dnf clean all
 
+# Install the latest non-preview powershell release.
+RUN LATEST_TAG=$(curl -L https://api.github.com/repos/powershell/powershell/releases/latest | jq -r '.tag_name') \
```

Comment:
Looks like you'll need to install `jq` as well.

Reply:
It is installed on the final image, maybe it was just there via a dependency. I'll ensure it's explicit.